docs/docs/how-tos/How to authenticate using Keycloak.md

```diff
@@ -0,0 +1,86 @@
+---
+id: login-keycloak
+title: How to authenticate using Keycloak and launch your server
+description: |
+  This is a how-to for authenticating using Keycloak.
+---
+
+# How to authenticate using Keycloak and launch your server
+
+This guide aims to give a basic overview of the Nebari login process. Your organization's Nebari deployment will likely have a slightly different process due to the many
+authentication providers available for Nebari can within Keycloak.
```

Comment:
Seems there is a word missing - integrate?

Reply:
you have a nice wording later on what is covered here - would suggest moving it here instead